optional_plugins/resultsdb/avocado_resultsdb/__init__.py

```diff
@@ -156,11 +151,13 @@ class ResultsdbResult(Result):
     description = 'Resultsdb result support'
 
     def render(self, result, job):
-        if (job.config.get('resultsdb_logs', None) is not None and
-                job.config.get('stdout_claimed_by', None) is None):
+        resultsdb_logs = job.config.get('plugins.resultsdb.logs_url')
+        stdout_claimed_by = job.config.get('stdout_claimed_by')
+        if (resultsdb_logs is not None and stdout_claimed_by is None):
             log_msg = "JOB URL    : %s/%s"
-            LOG_UI.info(log_msg, job.config.get('resultsdb_logs',
-                                                os.path.basename(job.logdir)))
+            LOG_UI.info(log_msg,
+                        resultsdb_logs,
+                        os.path.basename(job.logdir))
```

Comment:
This seems to be a code refactor, maybe it deserves its own commit.

Reply:
What part? I'm noot sure, just a small refactoring to fit in 80 columns using the new logic.

Reply:
Right, this is not related to this commit message. That is why I suggested to move it to another commit.

Reply:
Anyway, it does not have an impact on the module, so no problem keeping it this way.